python/GafferUITest/WidgetTest.py

```diff
@@ -415,7 +415,7 @@ def testWidgetAt( self ) :
 		w1.setPosition( imath.V2i( 100 ) )
 		w2.setPosition( imath.V2i( 300 ) )
 
-		self.waitForIdle( 10000 )
+		self.waitForIdle( 1000000 )
```

Comment:
As above, this is now taking the best part of 8 seconds. I'd really like to keep the test suite as fast to run as possible...

Reply:
Oh, I see these tests are the same ones that are disabled in a later commit. Could we just drop this commit then, to keep things quick on Linux?

Reply:
Also dropped in the latest push.